apps/prairielearn/elements/pl-code/pl-code.py

```diff
@@ -212,11 +213,20 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
                 f'Unknown language: "{language}". Must be one of the aliases listed in https://pygments.org/languages/, or the special language "ansi-color".'
             )
 
-    style = pl.get_string_attrib(element, "style", STYLE_DEFAULT)
+    # The `style` attribute is deprecated, but we still support it for backwards compatibility.
+    if pl.has_attrib(element, "style") and pl.has_attrib(element, "style-name"):
+        raise ValueError(
+            'Cannot use both "style" and "style-name" attributes at the same time.'
+        )
+    style_name = pl.get_string_attrib(
```

Comment:
I know this is a one liner but having a helper function that takes in the element variable and just returns the result would be nice.